src/main/java/org/jenkinsci/plugins/build_view_column/BuildViewColumnDescriptor.java

```diff
@@ -16,6 +16,10 @@ protected BuildViewColumnDescriptor() {
     }
 
     public static DescriptorExtensionList<BuildViewColumn,BuildViewColumnDescriptor> all() {
-        return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        try{
+            return Jenkins.getInstance().getDescriptorList(BuildViewColumn.class);
+        } catch(NullPointerException ex){
```

Comment:
You need not use `ExtensionList.lookup`, but the current code is very poor style. Explicitly check for null.